### DISCUSSION TITLE
消息事件unkown字段能否替换成该字段类型的默认值

### DISCUSSION THREAD

**Discussion by huisunan** [howmanybots/onebot#142] [问答]
![image](https://user-images.githubusercontent.com/30598579/102680861-509b1380-41f7-11eb-9e2e-55a08510734b.png)
      "sender": {
		"age": "unknown",
		"nickname": "unknown",
		"sex": "unknown",
		"user_id": 5711211111
	},
age是数字类型，返回的却是字符串

**Comment by kj415j45** [comment]
onebot目前仅为一个标准，具体到平台实现的问题请移步使用该标准的平台。

该问题似乎在 yyuueexxiinngg/onebot-kotlin@d377e3c616e988df0d4052bd3e998b49dce972ba 已尝试修复。但仍需要上游的 mamoe/mirai#234 完成。
